src/wmtk/autogen/edge_mesh/get_tuple_from_simplex_local_id.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <cassert>
+#include <wmtk/Tuple.hpp>
+#include "autogenerated_tables.hpp"
+#if !defined(_NDEBUG)
+#include "is_ccw.hpp"
+#endif
+
+namespace wmtk::autogen::edge_mesh {
+
+inline Tuple get_tuple_from_simplex_local_vertex_id(int8_t local_id, int64_t global_id = 0)
```

Comment:
Is it reasonable to have the global_id default to 0? I feel like having no default value makes more sense.